package/MDAnalysis/core/selection.py

```diff
@@ -802,10 +819,14 @@ class ProteinSelection(Selection):
     See Also
     --------
     :func:`MDAnalysis.lib.util.convert_aa_code`
+
+    .. versionchanged:: 2.0.0
```

Comment:
I think you'll need two empty lines here otherwise the versionchanged won't render properly.